\pdfminorversion=4
% utf-8
\documentclass[notes=hide]{beamer}
\usepackage[utf8]{inputenc}
\usepackage[english]{babel}
\usepackage{graphicx}
\usepackage{wasysym}
\usepackage{alltt}
\usepackage{bbm}
\usepackage{stmaryrd}
\usepackage{eurosym}
\usepackage{amssymb}
\usetheme{uds}
% \usepackage[dvipsnames]{xcolor}

\defbeamertemplate{headline}{page number}{%
	\vskip1pt%
	\setbeamertemplate{footline}[frame number]%
	\usebeamertemplate{footline}%
}

\setbeamertemplate{headline}[page number]
\setbeamerfont{smallfont}{size=\small}
\setbeamerfont{smallerfont}{size=\footnotesize}
\setbeamerfont{smallestfont}{size=\scriptsize}
\setbeamerfont{tinyfont}{size=\tiny}
\setbeamerfont{largefont}{size=\large}
\setbeamerfont{Largefont}{size=\LARGE}
%\setbeamerfont{page number in head/foot}{size=\tiny}
%\setbeamertemplate{headline}[frame number]


\newcommand\bluebox[2][uds@main]{{%
  \setlength{\fboxsep}{0pt}%
  \colorbox{#1}{#2\strut}%
}}

\def\Prob{\mbox{\bf P}}
\newcommand{\OO}{\mathcal{O}}
\newcommand{\N}{\mathbbm{N}}
\newcommand{\R}{\mathbbm{R}}
\newcommand{\len}[1]{{\vert #1 \vert}}
\newcommand{\emptystring}{\varepsilon}
\newcommand{\substr}[3]{#1[#2\ldots#3]}
\newcommand{\suffix}[2]{{#1}[{#2}\ldots]}
\newcommand{\prefix}[2]{{#1}[\ldots{#2}]}
\newcommand{\chr}[2]{#1[#2]}
\newcommand{\powset}[1]{2^{#1}}
\newcommand{\pos}{\ensuremath{\texttt{\upshape pos}}}
\newcommand{\lcp}{\ensuremath{\texttt{\upshape lcp}}}
\newcommand{\cld}{\ensuremath{\texttt{\upshape cld}}}
\newcommand{\rank}{\ensuremath{\texttt{\upshape rank}}}
\newcommand{\bwt}{\ensuremath{\texttt{\upshape bwt}}}
\newcommand{\bwtfind}{\ensuremath{\texttt{\upshape bwtfind}}}
\newcommand{\Occ}{\ensuremath{\texttt{\upshape Occ}}}
\newcommand{\less}{\ensuremath{\texttt{\upshape less}}}
\newcommand{\type}{\ensuremath{\texttt{\upshape type}}}
\newcommand{\lcpskip}{\ensuremath{\texttt{\upshape skip}}}
\newcommand{\gap}{\ensuremath{\text{--}}}
\newcommand{\iverl}{\llbracket}
\newcommand{\iverr}{\rrbracket}
\newcommand{\C}{\ensuremath{\texttt{\upshape C}}}
\newcommand{\T}{\mathcal{T}}
\newcommand{\I}{\mathcal{I}}
\newcommand{\F}{\mathcal{F}}
\newcommand{\X}{\mathcal{X}}
\newcommand{\A}{\mathcal{A}}
\newcommand{\RR}{\mathbb{N}}
\newcommand{\NN}{\mathbb{R}}
\newcommand{\W}{\mathcal{W}}


\newcommand{\blackboard}[1]{
\begin{block}<#1>{}
\begin{center}
\textbf{BLACK BOARD EXAMPLE}
\end{center}
\end{block}
}

\usepackage{listings} % ab Version 1.4 mit Python-Syntax-Highlighting!
\definecolor{mygray}{gray}{.50}
\lstset{language=Python,
    basicstyle=\scriptsize\ttfamily,
    stringstyle=\ttfamily\color{green},
    keywordstyle=\color{blue}\bfseries,
    commentstyle=\color{mygray},
    tabsize=4,
    numbers=left,
    numberstyle=\tiny,
    numbersep=5pt,
    morekeywords=assert,
    extendedchars=false,
    showstringspaces=false,
    frame=single
    }
\lstset{escapeinside={/*@}{@*/}}

\newcommand{\captionslide}[1]{
\begin{frame}
\frametitle{\phantom{NONE}}
\begin{center}
\vspace{1cm}
\usebeamerfont{Largefont}
          {\bf\em #1}
          \vspace{2cm}
\end{center}
\end{frame}
}
              
\title[Haplotype Assembly]{Computational Haplotyping}
%\author[SG]{\textit{Shilpa Garg}\\ PhD Student, MPI-Inf, Saarbr√ºcken, Germany \\with \\
%	Marcel Martin, Tobias Marschall }

\author[Shilpa Garg]{Shilpa Garg}
\date{March 6, 2018}
\institute{\scriptsize Max Planck Institute for Informatics, Germany \\ Center for Bioinformatics, Saarland University, Germany \\ Saarbr\"ucken Graduate School of Computer Science, Germany}


\begin{document}

\frame[plain]{\titlepage}

\begin{frame}{What is Haplotyping/Phasing?}
\begin{center}
\includegraphics[width=.8\textwidth]{figs/diploid-phasing-no-reads}
\end{center}
\begin{block}<2>{Applications}
\begin{itemize}
	\item Genome engineering
	\item Personalized medicine
	\item Human migration, evolutionary selection and population structure
\end{itemize}
\end{block}
\end{frame}

\begin{frame}{Approaches to Phasing}
\begin{center}
% \only<1>{\includegraphics[width=\textwidth]{figs/haplotyping-overview1}}%
\only<1>{\includegraphics[width=\textwidth]{figs/haplotyping-overview6}}%
\only<2>{\includegraphics[width=\textwidth]{figs/haplotyping-overview5}}%
\end{center}
\end{frame}

\begin{frame}{Combining Different Information Sources}
\begin{center}
\only<1>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections0}} %
\only<2>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections1}} % 
\only<3>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections2}} % 
\only<4>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections8}} % 
\only<5>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections9}} % 
% \only<4>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections3}} % 
\only<6>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections7}} % 
% \only<6>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections5}} % 
\end{center}
\end{frame}

% \begin{frame}{Challenges in Read-Based Phasing}
%  \begin{itemize}
%   \item Ordering of reads over the genome
%   \item Errorenous data
%  \end{itemize}
% \end{frame}

\begin{frame}{Read-Based Phasing (Single Individual)}
\begin{center}
\includegraphics[scale=.35]{figs/sih-phasing-complete2.pdf}
\end{center}
\end{frame}


\begin{frame}{Read-Based Phasing (Single Individual)}
\begin{center}
\includegraphics[scale=.35]{figs/sih-phasing-complete1.pdf}
\end{center}
\end{frame}

\begin{frame}{Read-Based Phasing (Single Individual)}
	\begin{center}
	%	\includegraphics[scale=.3]{figs/sih-phasing-complete.pdf}
		\includegraphics[scale=.35]{figs/sih-phasing-complete-haplo.pdf}		
	\end{center}
\end{frame}

\begin{frame}{Read-Based Phasing (Single Individual)}
\begin{center}
\begin{block}{Input: }
				\begin{itemize}
										\item \textcolor{cyan}{$g \in \{0,1,2\}^M$ : Input genotype for an individual }
					\item $\F \in \{0,1,-\}^{R\times M}$ : Input SNP matrix for an individual 

				\end{itemize}
	\end{block}
	\bigskip
	\includegraphics[scale=0.35]{figs/sih-phasing-complete-matrix.pdf}%
%width=0.85\textwidth
\end{center}
\end{frame}

\begin{frame}{Read-Based Phasing (Single Individual)}
	\begin{center}
\begin{block}{Input:}
		\begin{itemize}
		\color{blue}	\item $W \in \RR^{R \times M}$ : Matrix of confidence for each entry.

		\end{itemize}
	\end{block}
		\bigskip
		\bigskip
         \vspace*{1.9mm}
	\only<1>{\includegraphics[scale=0.35]{figs/sih-phasing-complete-phred0}}
		\only<2>{\includegraphics[scale=0.35]{figs/sih-phasing-complete-phred}}
%width=.9\textwidth
\end{center}
\end{frame}

\begin{frame}{Read-Based Phasing (Single Individual)}
	\vspace*{1.3mm}
	%\bigskip
	\begin{center}
		\begin{block}{Goal:}
			\begin{itemize}
				\item Conflict free matrix for an individual				
			\end{itemize}
		\end{block}
		\bigskip
		\bigskip
		\vspace*{2.23mm}
		\includegraphics[scale=0.35]{figs/sih-phasing-complete4.pdf}
		%width=.9\textwidth
	\end{center}
\end{frame}

\begin{frame}{Read-Based Phasing (Single Individual)}
	%\vspace*{1.3mm}
	%\bigskip
	\begin{center}
	\begin{block}{Objective function: \color{red}wMEC }

		Flip the set of entries with minimum cost such that a conflict-free bipartition of rows exist.
		\begin{center}
			minimize $\sum_{(j,k)\in E}W(j,k)$
		\end{center}
		
	\end{block}
		\bigskip
		%\bigskip
		\vspace*{.6mm}
		\includegraphics[scale=0.35]{figs/sih-phasing-complete4.pdf}\\
		\begin{center}
			\includegraphics[width=.9\textwidth]{figs/sih-phasing-complete-haplo-goal1}
		\end{center}
			
		%width=.9\textwidth
	\end{center}
\end{frame}

%\begin{frame}{Read-Based Phasing (Single Individual)}
%\begin{center}
%\only<1->{
%	\begin{block}{Objective \color{red}wMEC }
%							Flip the entries with minimum cost such that a conflict-free bipartition of rows exist.
%		\begin{center}
%	minimize $\sum_{(j,k)\in E}\mathcal{W}(j,k)$
%		\end{center}
%		
%	\end{block}}	
%		%\visible<2>{
%
%				\includegraphics[width=.9\textwidth]{figs/sih-phasing-complete-haplo-goal}
%								\begin{block}{Goal:}
%									\begin{itemize}
%										\item $h^{0}, h^{1} \in \{0,1\}^{M}$: Haplotypes for an individual
%									\end{itemize}	
%								\end{block}
%	%}%
%\end{center}
%\end{frame}


\begin{frame}{WhatsHap Algorithm (Sketch)}
% 		\begin{block}{}
% 			Introduced at RECOMB 2014
% 		\end{block}
\begin{block}{}
\emph{Approach:} FPT algorithm with ``coverage'' as parameter.
\begin{itemize}
% \item \emph{coverage:} number of non-dash (i.e.~0,1) positions in a column
 \item Can be bounded without loosing relevant information
 \item Dynamic Programming
\end{itemize}
\end{block}
% \begin{center}
% \includegraphics[width=0.5\textwidth]{figs/phasing-problem5}
% \end{center}
% \vspace{-2em}
%\begin{block}<2->{Dynamic Programming}
%Proceed \emph{column-wise}. For each column:
% \begin{enumerate}
%  \item \emph{Enumerate all bipartitions} of rows that cover that column
%  \item Compute number of bit-flips incurred by each bipartition
%  \item Project costs to ``intersection column''
% \end{enumerate}
%\end{block}
\end{frame}

%\begin{frame}{Runtime}
%\begin{block}{Notation}
%\begin{itemize}
% \item \emph{$c$: maximum coverage}, i.e.\ maximum number of active reads per column
% \item \emph{$M$: number of columns}, i.e.\ number of variants to be phased
%\end{itemize}
%\end{block}
%\begin{block}{Runtime}
%\[O(2^c\cdot M)\]
%\end{block}
%\begin{block}{Conclusions}
%\begin{itemize}
% \item Linear in the number of variants (good)
% \item Independent of read length (good, especially for long reads)
% \item Exponential in the coverage (no big problem in practice, can prune to, say, 15$\times$)
%\end{itemize}
%\end{block}
%\end{frame}

\captionslide{Read-Based Phasing of\\ Related Individuals\\
{ \begin{itemize}
 \scriptsize \color{red}
		\item How much coverage is needed for resolving haplotypes in related individuals as opposed to single individual?
		\item Can we get long-range haplotypes for all the individuals?
	\end{itemize} }
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
	}


\begin{frame}{Phasing Related Individuals}

\begin{center}
	\only<1>{\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete1}\vspace*{2.05cm}}%
	\only<2>{\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete2}\vspace*{2.05cm}}%
	\only<3>{\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete3}
		\begin{block}{}
			\begin{itemize}
				\item \emph{Genetic haplotyping:} cannot phase SNP 3 (all heterozygous)
				\item \emph{Read-based haplotyping:} cannot connect SNP 4 in child
				\item Solution: \emph{combine genetic and read-based haplotyping}
			\end{itemize}
		\end{block}}%
%\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete}
\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

\begin{frame}{Phasing Related Individuals}
	\begin{center} 
		\only<1>{\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete}}%
		%\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete}

		\begin{block}{}
			\begin{itemize}
				\item \emph{Genetic haplotyping:} cannot phase SNP 3 (all heterozygous)
				\item \emph{Read-based haplotyping:} cannot connect SNP 4 in child
				\item Solution: \emph{combine genetic and read-based haplotyping}
			\end{itemize}
		\end{block}
			\end{center}
% 					\vspace*{4.5mm}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
					\end{frame}


%\begin{frame}{Phasing Related Individuals}
%\begin{block}{Input:}
%\begin{itemize}
% %\item $\I$ : Set of individuals, $\T$ : Set of trio relationships
%  \item $g_i \in \{0,1,2\}^M$ : Input genotype for individual $i$
%			 \item $\F_i \in \{0,1,-\}^{R_i\times M}$ : Input SNP matrix for individual $i$ 
%
% \end{itemize}
%\end{block}
%\begin{center}
%\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-matrix}
%\end{center}
%\end{frame}
%
%
%
%\begin{frame}{Phasing Related Individuals}
%	\begin{block}{Input:}
%		\begin{itemize}
%
%			\item $\W_i \in \RR^{R_i \times M}$ : Matrix of phred scores for individual $i$
%			%\item $\A(k)$ : Set of reads active in column $k$
%		\end{itemize}
%	\end{block}
%	\vspace*{5.5mm}
%	\begin{center}
%		\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-phred}
%	\end{center}
%\end{frame}
%
%\begin{frame}{Phasing Related Individuals}
%	\begin{block}{Input:}
%		\begin{itemize}
%			\item $\X \in \RR^M$ : Recombination cost vector
%			\item $t_{m\rightarrow c}, t_{f\rightarrow c} \in \{0,1\}^M$ : transmission vectors for trio
%		\end{itemize}
%	\end{block}
%	\begin{center}
%		\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-recomb-vector}
%	\end{center}
%\end{frame}

%\begin{frame}{Phasing Related Individuals}
%	\begin{block}{Goal:}
%		\begin{itemize}
%			\item Conflict free matrix by following Mendelian Inheritance.
%			\item Meiotic Recombination
%		\end{itemize}
%	\end{block}
%	\vspace*{1.4mm}
%	\begin{center}
%		\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-recomb-vector}
%	\end{center}
%\end{frame}

%\begin{frame}{Phasing Related Individuals}
%	\begin{block}{}
%		\begin{itemize}
%			\item \emph{Genetic haplotyping:} cannot phase SNP 3 (all heterozygous)
%			\item \emph{Read-based haplotyping:} cannot connect SNPs 3-5 in child
%			\item Solution: \emph{combine genetic and read-based haplotyping}
%		\end{itemize}
%	\end{block}
%	\begin{center}
%		\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-error}
%	\end{center}
%\end{frame}

\begin{frame}{From wMEC to ``MEC on Pedigrees'' (PedMEC)}
	\begin{center}
		\only<1>{\includegraphics[width=\textwidth, height=175]{figs/pedmec1}}%
		\only<2>{\includegraphics[width=\textwidth, height=175]{figs/pedmec2}}%
		\only<3>{\includegraphics[width=\textwidth, height=175]{figs/pedmec3}}%
		\only<4>{\includegraphics[width=\textwidth, height=175]{figs/pedmec4}}%
			%	\only<5>{\includegraphics[width=\textwidth, height=175]{figs/pedmec5}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
	\end{frame}

%\begin{frame}{From wMEC to ``MEC on Pedigrees'' (PedMEC)}
%			\begin{block}{Objective function (minimize):\color{red}PedMEC }
%				\scriptsize
%				\begin{itemize}
					%\item[] \[\sum_{i\in\mathcal{I}}\sum_{(j,k)\in E_i}\mathcal{W}_i(j,k)+\sum_{(m,f,c)\in\mathcal{T}}\mathcal{X}(t_{m\to c}) + \mathcal{X}(t_{f\to c})\] 
				%\end{itemize}
		%		
		%	\end{block}

		%	\visible<2->{\begin{center}
		%\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-haplotype3}
	%\end{center}}
	%			\visible<3->{\begin{center}
	%					\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-haplotype2}
	%				\end{center}}
	%							\visible<4->{\begin{center}
	%									\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-haplotype1}
	%								\end{center}}
	%											\begin{center}
	%													\includegraphics[width=.9\textwidth]{figs/trio-phasing-complete-haplotype}
	%												\end{center}
	%												\begin{block}{Goal:}
	%													\begin{center}
	%														\scriptsize  $h^{0}_{i}, h^{1}_{i} \in \{0,1\}^{M}$: Haplotypes for individual $i$
	%													\end{center}	
	%												\end{block}
		%\visible<6->{\begin{block}{Goal:}
		%	\begin{center}
		%		\scriptsize  $h^{0}_{i}, h^{1}_{i} \in \{0,1\}^{M}$: Haplotypes for individual $i$
		%	\end{center}	
		%\end{block}}
%\end{frame}

\begin{frame}

		\begin{block}{WhatsHap (Related Individuals): \color{red} FPT }
			\begin{itemize}
				\item Use similar approach as for wMEC
				\item Add additional dimension to DP table: \emph{transmission status}\\
				i.e.\ which haplotype in each parent is transmitted to child
			\end{itemize}
		\end{block}
		\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

% \begin{frame}{WhatsHap (Related Individuals)}
% 	\begin{center}
% 		\only<1>{\includegraphics[width=\textwidth, height=175]{figs/pedmeccost2}}%
% 		\only<2>{\includegraphics[width=\textwidth, height=175]{figs/pedmeccost1}}%
% % 		\only<3>{\includegraphics[width=\textwidth, height=175]{figs/pedmeccost3}}%
% % 		\only<4>{\includegraphics[width=\textwidth, height=175]{figs/pedmeccost4}}%
% %		\only<5>{\includegraphics[width=\textwidth, height=180]{figs/pedmec5}}%
% 	\end{center}
% 	\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
% \end{frame}

%\begin{frame}{WhatsHap (Related Individuals) }
%%	\begin{block}{WhatsHap (Related Individuals) }
%%                \scriptsize
%%		\textcolor{red}{DP Initialization}
%%		\begin{equation}\label{eqn:delta_c}
%%		\Delta_C(k,B,t)= \min_{a\in \{0,1\}^{\mathcal{S}(k,B,t)}}\left\{\sum_{S\in\mathcal{S}(k,B,t)}W_{k,S}^{a(S)}\right\},
%%		\end{equation}
%%	\end{block}
%%        \vspace*{6mm}
%
%		\begin{columns}[]
%			 \hspace*{10mm}
%			\begin{column}{0.4\textwidth}
%				\includegraphics[scale=0.15]{figs/trio-phasing-complete-first-column}
%                                % width=0.9\textwidth
%                                \vskip5mm
%                                \vspace*{3mm}
%			\end{column}
%		\only<1>{                         \hspace*{1mm}
%				\begin{column}{0.9\textwidth}
%					\includegraphics[height=.5\textwidth]{figs/trio-phasing-complete-cost-computation1-first-bipartition}
%                   \vskip5mm           
%			\end{column} 
%			\vskip5mm}
%			\only<2>{	     \hspace*{.2mm}
%						\begin{column}{0.9\textwidth}
%							%\begin{center}
%							\includegraphics[height=.5\textwidth]{figs/trio-phasing-complete-cost-computation1-second-transmission}
%                                                        % width=0.5\textwidth, height=0.9\textwidth
%							%\end{center}
%
%						\end{column}
%						\vskip5mm
%						 %\vspace*{.1cm}
%						}
%						%\vskip{5mm}
%		\end{columns}
%
%	
%\end{frame}

% \begin{frame}
% 	\begin{block}{WhatsHap (Related Individuals)}
%                \scriptsize
% 		\textcolor{red}{DP Recurrence}
% 		\begin{align}\label{eqn:recurrence}
% 			& C(k+1,B,t)= \Delta_C(k+1,B,t)\\
% 			& \quad + \min_{\substack{B'\in\mathcal{B}(A(k)):B'\simeq B\\t'\in\{0,1\}^{2 {\mathcal{T}}}}}\big\{C(k,B',t')+d_H(t,t')\cdot\mathcal{X}(k+1)\big\}, \nonumber
% 		\end{align}
% 	\end{block}
% %	\begin{center}
% \only<1>{		\begin{columns}[]
% 			                       \hspace*{10mm}
% 
% 			\begin{column}{0.4\textwidth}
% 				\includegraphics[scale=0.15]{figs/trio-phasing-complete-scond-column}
%                                % width=0.9\textwidth
% 				 \begin{block}{}
% 				 	\tiny $C(2,B,t)=0+ \min\{4 \ldots        \}$
% 				 \end{block}
% 
% 			\end{column}
%                        \hspace*{2mm}
% 			\begin{column}{0.9\textwidth}
% 				
% 				%\begin{center}
% 				%\includegraphics[scale=0.3]{figs/trio-phasing-complete-cost-computation1-second-transmission}
%                                %width=0.5\textwidth, height=0.9\textwidth
% 				\includegraphics[height=.5\textwidth]{figs/trio-phasing-complete-cost-computation1-second-column-dp-value}\\
% 				%\end{center}
% 				
% 			\end{column}
% 		\end{columns}
% 	}
% \only<2>{\begin{columns}[]
% 	\hspace*{10mm}
% 	\begin{column}{0.4\textwidth}
% 		\vspace*{1.8mm}
% 		\includegraphics[scale=0.15]{figs/trio-phasing-complete-scond-column}
% 		% width=0.9\textwidth
% 		\begin{block}{}
% 			\tiny $C(2,B,t)=0+ \min\{4, 2 + 1\cdot 20, 4 + 1\cdot 20, 2 + 2\cdot20\}$
% 		\end{block}
% 		
% 	\end{column}
% 	\hspace*{2mm}
% 	\begin{column}{0.9\textwidth}
% 		
% 		%\begin{center}
% 		%\includegraphics[scale=0.3]{figs/trio-phasing-complete-cost-computation1-second-transmission}
% 		%width=0.5\textwidth, height=0.9\textwidth
% 		\includegraphics[height=.5\textwidth]{figs/trio-phasing-complete-cost-computation1-second-column-dp-value2}\\
% 		%\end{center}
% 		
% 	\end{column}
% \end{columns}
% 	
% 	}
% 		
% %	\end{center}
% 	
% \end{frame}


\begin{frame}{}
	\begin{block}<1->{Runtime}
		Two extra bits for each mother-father-child relationship:
\[O(2^{2t+c}\cdot M)\qquad\qquad\text{(hiding some ugly terms)}\]
		where $c$ is the \emph{maximum of sum of coverages across individuals} and $t$ is the \emph{number of mother-father-child relationship}.
	\end{block}
% 	\begin{block}<2>{}
% 		Still feasible for, e.g., $t=1$ and $c=3\cdot 5$. 
% 		\emph{Linear} in the number of variants.
% 		\emph{Independent} of read length.
% 	\end{block}
	\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

%\begin{frame}{Datasets}
%	\begin{itemize}
%		\item \emph{Real Dataset:} AJ trio from Genome in a Bottle Consortium.
%				\begin{itemize}
%					\item \emph{Ground Truth unphased genotypes}
%					\item \emph{Ground Truth phased variants}
%
%					\item PacBio dataset
%					\item 10X Genomics
%				\end{itemize}
%		\item \emph{Simulated Dataset}
%				\begin{itemize}
%					\item Virtual Child
%					\item Simulated PacBio dataset
%				\end{itemize}
%	\end{itemize}
%\end{frame}
\begin{frame}{Datasets}
	\begin{center}
		\includegraphics[scale=0.1]{figs/flowdiagram1}
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

%\begin{frame}
%	\begin{block}{Algorithm: }
%		\textcolor{red}{DP Recurrence}
%		\begin{align}\label{eqn:recurrence}
%		& C(k+1,B,t)= \Delta_C(k+1,B,t)\\
%		& \quad + \min_{\substack{B'\in\mathcal{B}(A(k)):B'\simeq B\\t'\in\{0,1\}^{2\abs{\mathcal{T}}}}}\big\{C(k,B',t')+d_H(t,t')\cdot\mathcal{X}(k+1)\big\}, \nonumber
%		\end{align}
%		\begin{definition}[Bipartition compatibility]
%			Let $B=(P,Q)$ be a bipartition of $A$ and $B'=(P',Q')$ be a bipartition of $A'$.
%			We say that $B$ and $B'$ are \emph{compatible}, written $B\simeq B'$, if
%			$P\cap(A\cap A') = P'\cap(A\cap A')$
%			and
%			$Q\cap(A\cap A') = Q'\cap(A\cap A')$.
%		\end{definition}
%	\end{block}
%		\includegraphics[width=.3\textwidth]{figs/trio-phasing-complete-first-column}
		%\includegraphics[width=.3\textwidth]{figs/trio-phasing-complete-cost-computation1}
%\end{frame}




\begin{frame}{Phasing Related Individuals (Performance)}
\begin{center}
%\only<1>{\includegraphics[scale=.23]{figs/whatshap-pedmec-performance1}} % 
\only<1>{\includegraphics[scale=.23]{figs/whatshap-pedmec-performance2}} %
\only<2>{\includegraphics[scale=.1]{figs/flowdiagram2}} %
\only<3>{\includegraphics[scale=.23]{figs/whatshap-pedmec-performance3}} %
% \only<4>{\includegraphics[scale=.1]{figs/flowdiagram}} %
\end{center} %\vspace{-.5em}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

% \begin{frame}{Comparison to other methods}
% \begin{columns}
% \begin{column}{.55\textwidth}
% \begin{block}{Three \emph{independent} phasings}
% \begin{itemize}
%  \item PedMEC on 15$\times$ PacBio data
%  \item 10xGenomics synthetic long-read phasing
%  \item Statistical phasing by SHAPEIT using 1000 Genomes panel
% \end{itemize}
% \end{block}
% \end{column}
% \begin{column}{.45\textwidth}
% \begin{center}
% \only<1>{\includegraphics[scale=0.23]{figs/giab-threeway-left}}%
% %\only<2>{\includegraphics[width=\columnwidth]{figs/giab-threeway-both}}%
% \end{center}
% \end{column}
% \end{columns}
% \let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
% \end{frame}

\begin{frame}{Phasing Across Block Boundaries}

	\begin{columns}[]
			\begin{column}{.1\textwidth}
				
			\end{column}
		\begin{column}{0.8\textwidth}
%\begin{center}
		\includegraphics[height=.8\textwidth, width=10\textwidth, keepaspectratio]{figs/long-haplotypes}
%\end{center}
			
			\end{column}
       \hspace*{-2.5cm}
	\begin{column}{.48\textwidth}
			\begin{block}{}
				\emph{Block} is connected component of reads.
				
			\end{block}
	\begin{block}{}
			\begin{itemize}
				\item Phase $99\%$ of the total heterozygous SNPs.
				\item Phasing error rate of $1\%$.
			\end{itemize}
			
		\end{block}
		
	\end{column}
	\begin{column}{\textwidth}

	\end{column}
		\end{columns}
%	\begin{center}
	
%	\end{center}
%	\begin{block}{}
	
%		\begin{itemize}
%			\item Previous slide: no connection by read $\rightarrow$ count as \emph{unphased}
%			\item But there is information \emph{across block boundaries:}
%			\begin{itemize}
%				%\item Phase \emph{between} blocks ``guessed'' correctly in 90\% of all cases.
%				\item Phase $99\%$ of the total heterozygous SNPs in the largest block.
%				\item Phasing error rate of $1\%$.
%			\end{itemize}
%		\end{itemize}
%	\end{block}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

\begin{frame}{Conclusion}
	\begin{itemize}
		\item Formalization of PedMEC problem. 
		\item Unified framework for integrative read-based phasing and genetic haplotyping.
		\item More accurate and complete phasing with family relationship at low coverages of $2\times$ as compared to single individual at $15\times$.
		\item Produces chromosome-length haplotypes by merging the blocks.
% 		\item \emph{Future plan} is to integrate statistical, genetic and read-based phasing.
	\end{itemize}
% 	\begin{block}{}
% 		\emph{Technology Track talk} by Marcel on WhatsHap software on Tuesday at 2:20 pm in the room Northern Hemisphere E3/E4
% 	\end{block}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Martin M, Marschall T. \textit{Read-based phasing of related individuals}. Bioinformatics. 2016}}
\end{frame}

\captionslide{Back to the big picture}
% \begin{frame}{Combining Different Information Sources}
% \begin{center}
% \only<1>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections0}} %
% \only<2>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections1}} % 
% \only<3>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections2}} % 
% \only<4>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections8}} % 
% \only<5>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections9}} % 
% % \only<4>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections3}} % 
% \only<6>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections7}} % 
% % \only<6>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections5}} % 
% \end{center}
% \end{frame}
\begin{frame}{Combining Different Information Sources}
	\begin{center}
		\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections7}
	\end{center}

\end{frame}

\captionslide{Dense and accurate whole-chromosome haplotyping of individual genomes \\
{ \begin{itemize}
 \scriptsize \color{red}
		\item How much coverage is required for each sequencing technology in an integrative approach?
	\end{itemize} }
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}
}

\begin{frame}{StrandSeq in the MEC Framework}
\begin{center}
\includegraphics[width=\textwidth]{figs/strandseq}
\end{center}
\begin{block}{}
\begin{itemize}
 \item StrandSeq data leads to wMEC problem with \emph{two rows for each cell}, one for Watson, one for Crick
 \item \emph{Additional constraint} that Watson/Crick reads do not come from same haplotype
 \item Solving it exactly is infeasible (with known algorithms)
\end{itemize}
\end{block}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}
\end{frame}

\begin{frame}{Combining StrandSeq and PacBio}
\begin{center}
\includegraphics[width=\textwidth]{figs/strandseq-plus-pacbio}
\end{center}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}
\end{frame}

% \begin{frame}{Combining 10X Genomics and PacBio}
% \begin{center}
% \includegraphics[width=\textwidth]{figs/10xg-plus-pacbio}
% \end{center}
% \end{frame}


\begin{frame}{Integrative phasing approach}
\begin{block}{}
\begin{itemize}
%  \item Long-range haplotyping using StrandSeq or 10xGenomics.
%  \item Combine with long-read sequencing technologies like PacBio or ONT
 \item In WhatsHap algorithm, solve paramerterized algorithm using Dynamic Programming (DP)
 \item WhatsHap solves MEC optimaly for bounded coverage
\end{itemize}
\end{block}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}
\end{frame}


\begin{frame}{Datasets}
\begin{itemize}
 \item Illumina: 1000 Genome Project Consortium
 \item PacBio: Genome in a Bottle Consortium
 \item Pre-assembled 10X Genomics haplotypes: 10x Genomics website
 \item StrandSeq: publicly available at ENA
 \item Gold-standard phasing: Illumina platinum haplotypes from a pedigree
\end{itemize}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}

\end{frame}

\begin{frame}{Results (Phasing efficacy)}
\begin{center}
\includegraphics[width=\textwidth, height=160]{figs/Figure1}
\end{center}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}

\end{frame}


\begin{frame}{Comprehensive coverage analysis for haplotype quality}
\center{Completeness} 
\begin{center}
\includegraphics[width=\textwidth]{figs/Figure3_1}
\end{center}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}

\end{frame}

\begin{frame}{Comprehensive coverage analysis for haplotype quality}
\center{Accuracy}
\begin{center}
\includegraphics[width=.9\textwidth, height = 144]{figs/Figure3_5}
\end{center}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}

\end{frame}

\begin{frame}{Comprehensive coverage analysis for haplotype quality}
\center{Contiguity}
\begin{center}
\includegraphics[width=\textwidth]{figs/Figure3_2}
\end{center}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}

\end{frame}



\begin{frame}{Genome-wide phasing of a single individual (NA12878)}
\begin{center}
\includegraphics[width=\textwidth]{figs/Figure4_1}
\end{center}
\let\thefootnote\relax\footnote{\tiny\textcolor{gray}{Porubsky D*, \textbf{Garg S*}, Sanders AD*, Korbel J, Guryev V, Lansdorp PM, Marschall T. \textit{Dense and accurate whole-chromosome haplotyping of individual genomes}. Nature Communications. 2017}}

\end{frame}

\captionslide{Back to the big picture}

\begin{frame}{Combining Different Information Sources}
	\begin{center}
		\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections7}
	\end{center}
\end{frame}

% \begin{frame}{Combining Different Information Sources}
% \begin{center}
% \only<1>{\includegraphics[width=\textwidth]{figs/haplotyping-overview-connections5}} % 
% \end{center}
% \end{frame}

\begin{frame}{}
\begin{center}
\includegraphics[scale=.35]{figs/sih-phasing-complete-assembly.pdf}
\end{center}
\begin{itemize}
 \item Reference bias
\end{itemize}

\end{frame}

\begin{frame}{}
\begin{center}
\includegraphics[scale=.35]{figs/sih-phasing-complete-assembly1.pdf}
\end{center}
\begin{itemize}
 \item Goal: two genome sequences -- haplotypes.
\end{itemize}
\end{frame}

\captionslide{Haplotype-aware denovo assembly \\
 {\begin{itemize}
 \scriptsize \color{red}
		\item What is the cost effective way of generating the good quality assemblies?
		\item Can we phase even large structural variants over the genome?
	\end{itemize} }
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
}


\begin{frame}{Haplotype-aware denovo assembly}
\begin{itemize}
 \item Port it as a graph problem -- assembly graph.
\end{itemize}
	\begin{center}
		\only<1>{\includegraphics[width=\textwidth, height=160]{figs/assembly4}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Diploid genome assembly}
\begin{itemize}
 \item Graph construction
\end{itemize}

	\begin{center}
		\only<1>{\includegraphics[width=.8\textwidth, height=160]{figs/assembly_graphs}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Related work}
\begin{itemize}
 \item Falcon Unzip performs contig-based phasing based on PacBio data
\end{itemize}
	\begin{center}
		\only<1>{\includegraphics[width=.8\textwidth, height=155]{figs/cropped_falcon-unzip}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Why contig-based phasing is not complete?}
\begin{itemize}
 \item Phasing SNPs by aligning PacBio reads to contigs
\end{itemize}
	\begin{center}
		\only<1>{\includegraphics[width=.8\textwidth, height=155]{figs/ex_graph_approach}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Diploid genome assembly pipeline}
	\begin{center}
		\only<1>{\includegraphics[width=\textwidth, height=175]{figs/pipeline}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}


\begin{frame}{Diploid genome assembly pipeline}
\begin{itemize}
 \item Long read-alignments to graph
\end{itemize}

	\begin{center}
		\only<1>{\includegraphics[width=\textwidth, height=155]{figs/assembly9}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Diploid genome assembly pipeline}
\begin{itemize}
 \item Detect bubble chains
\end{itemize}

	\begin{center}
		\only<1>{\includegraphics[width=\textwidth, height=155]{figs/assembly6}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Diploid genome assembly pipeline}
\begin{itemize}
 \item Phasing bubble chains
\end{itemize}

	\begin{center}
		\only<1>{\includegraphics[width=\textwidth, height=155]{figs/assembly5}}%
	\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Phasing bubble chains}
 \begin{itemize}
  \item Observation: Phasing bubble chains is similar to phasing multi-allelic variants
  \item Extend MEC formulation for bi-allelic variants to multi-allelic (gMEC)
  \item Solve in a WhatsHap style Dynamic Programming (DP) algorithm
  \item WhatsHap solves gMEC optimally for bounded coverage 
 \end{itemize}
 \let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Datasets}
\begin{itemize}
 \item Pseudo-diploid (SK1+Y12) yeast genome
 \item Illumina and PacBio data: publicly available (ENA)
 \item Gold-standard assemblies generated using high coverage PacBio data
\end{itemize}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Assembly Accuracy}
\begin{center}
\begin{table}
\begin{tabular}{ |l|c|c|c| } 
 \hline
 Statistics & PacBio & Graph-based  & Falcon Unzip \\ 
 & coverage &  approach &  \\ 
  \hline
 Average Identity[\%] & 10$\times$& 99.50 & \textemdash\\
   & 20$\times$& 99.61  &\textemdash\\
   & 30$\times$& 99.80 &99.4 \\
 Phasing error rate[\%] & 10$\times$& 2.5 & \textemdash\\
   & 20$\times$& 1.5  &\textemdash\\
   & 30$\times$& 0.7 & 3.8 \\
   \hline
\end{tabular}
\\[10pt]
 \caption{Comparison of two phasing methods, Falcon Unzip and our graph-based approach, at different PacBio coverage levels.}
\end{table}
\label{table1}
\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}


\begin{frame}{Assembly Contiguity}
\begin{center}
\begin{table}
\begin{tabular}{ |l|c|c|c| } 
 \hline
 Statistics & PacBio & Graph-based  & Falcon Unzip \\ 
 & coverage &  approach &  \\ 
  \hline
   N50 haplotig size [bp]& 10$\times$& 40k &\textemdash\\
   & 20$\times$& 42k &\textemdash\\
   & 30$\times$& 43k &32k\\ 
   \hline
\end{tabular}
\\[10pt]
 \caption{Comparison of two phasing methods, Falcon Unzip and our graph-based approach, at different PacBio coverage levels.}
\end{table}
\label{table2}
\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}

\begin{frame}{Assembly Completeness}
\begin{center}
\begin{table}
\begin{tabular}{ |l|c|c|c| } 
 \hline
 Statistics & PacBio & Graph-based  & Falcon Unzip \\ 
 & coverage &  approach &  \\ 
  \hline
  Haplotig size [Mbp] & 10$\times$& 20.7 &\textemdash\\
   & 20$\times$& 23.1 &\textemdash\\
   & 30$\times$& 23.9 & 16.6\\
   \# Unphased contigs  & 10$\times$& 2 &\textemdash\\
   & 20$\times$& 2 &\textemdash\\
   & 30$\times$& 2 &77\\
 \hline
\end{tabular}
\\[10pt]
 \caption{Comparison of two phasing methods, Falcon Unzip and our graph-based approach, at different PacBio coverage levels.}
\end{table}
\label{table3}
\end{center}
\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}


\begin{frame}{Phasing the Structural Variations}
\begin{center}
		\only<1>{\includegraphics[width=.6\textwidth, height=155]{figs/wh}}%
	\end{center}
	\let\thefootnote\relax\footnote{\scriptsize\textcolor{gray}{\textbf{Garg S}, Rautiainen M, Novak A, Durbin R \& Marschall T. \textit{A graph-based approach to diploid genome assembly}. Submitted}}
\end{frame}
% \captionslide{Efficient approach for statistical and read-based phasing}
% 
% \begin{frame}{Workflow}
% 	\begin{center}
% 		\only<1>{\includegraphics[width=\textwidth, height=175]{figs/pipeline}}%
% 	\end{center}
% \end{frame}

\begin{frame}{Conclusion}
	\begin{itemize}
		\item Developed novel haplotype-aware denovo assembly pipeline
		\item Integrative pipeline produces good quality haplotypes even at low coverage of 10$\times$
		\item Phase even large structural variants
	\end{itemize}
	\begin{block}{}
		\emph{Summary} Developed efficient algorithms for haplotyping by combining different information sources and demonstrated these methods on real genomes.
	\end{block}
\end{frame}

% \begin{frame}{Acknowledgements}
% Richard Durbin \\
% Benedict Paten \\
% Jan Korbel \\
% Peter M Lansdorp \\
% David Porubsky \\
% Thomas Lengauer\\
% Tobias Marschall \\
%  
% \end{frame}


\captionslide{Thanks \\ \small Questions?}





\end{document}
